### PR TITLE
8400 cbyrd remove references to arches collector

### PR DESCRIPTION
**Pull request by chrabyrd**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

Removes all references to Arches collector

### Issues Solved

#8400 

### Further comments

- Some migrations needed to be appended with `8528_` to work correctly
- APITests.test_api_base_view has been disabled
- some 6.1.x changes that aren't currently on 7.0.x were pulled into this branch

### CODE REVIEW

File: tests/views/api_tests.py [hunk 1]
```diff
@@ -74,24 +74,24 @@ def setUpClass(cls):
         test_pkg_path = os.path.join(test_settings.TEST_ROOT, "fixtures", "testing_prj", "testing_prj", "pkg")
         management.call_command("packages", operation="load_package", source=test_pkg_path, yes=True)
 
-    def test_api_base_view(self):
```
Comment (chiatt) [inline tests/views/api_tests.py:77]: I think the mobilesurvey endpoint could just be swapped out for another endpoint.  The test passed for me using `api_node_value`.


```
    def test_api_base_view(self):
        """
        Test that our custom header parameters get pushed on to the GET QueryDict

        """
        factory = RequestFactory(HTTP_X_ARCHES_VER="2.1")
        view = APIBase.as_view()

        request = factory.get(reverse("api_node_value", kwargs={}), {"ver": "2.0"})
        request.user = None
        response = view(request)
        self.assertEqual(request.GET.get("ver"), "2.0")

        request = factory.get(reverse("api_node_value"), kwargs={})
        request.user = None
        response = view(request)
        self.assertEqual(request.GET.get("ver"), "2.1")
        ```

File: tests/fixtures/testing_prj/testing_prj/settings.py [hunk 2]
```diff
@@ -130,8 +130,7 @@
 # Identify the usernames and duration (seconds) for which you want to cache the time wheel
 CACHE_BY_USER = {"anonymous": 3600 * 24}
 
-MOBILE_OAUTH_CLIENT_ID = ""  # '9JCibwrWQ4hwuGn5fu2u1oRZSs9V6gK8Vu8hpRC4'
-MOBILE_DEFAULT_ONLINE_BASEMAP = {"default": "mapbox://styles/mapbox/streets-v9"}
+OAUTH_CLIENT_ID = ""  # '9JCibwrWQ4hwuGn5fu2u1oRZSs9V6gK8Vu8hpRC4'
```
Comment (chiatt) [inline tests/fixtures/testing_prj/testing_prj/settings.py:133]: The release notes should mention that this setting has been changed. Admins will have to modify it in their projects.

File: arches/app/media/css/arches.scss [hunk 3]
```diff
@@ -11969,28 +11969,6 @@ a.search-facet-item.disabled {
...
-
-.mobile-project-manager-editor .library-card:hover {
-    background: #fafafa;
-}
-
-.mobile-project-manager-editor .card-nav-container {
-    margin-bottom: 0;
-}
-
 .mpm-project-card {
```
Comment (chiatt) [inline arches/app/media/css/arches.scss:11972]: `.mpm` was an abbreviation for 'mobile-project-manager', so those classes can probably be removed as well (hopefully they weren't used anywhere else.

**Review by chiatt — CHANGES_REQUESTED** [review]
Hey @chrabyrd - This looks really good. There are just a few are just a few minor changes that I think would be worth making.

**Comment by chrabyrd** [comment]
@chiatt this should be g2g now

**Review by chiatt — APPROVED** [review]

**Pull request merged**